lib/src/cross_platform/native.dart

```diff
@@ -19,19 +19,36 @@ int _sizeOfEntity(FileSystemEntity entity) {
   return stat.size;
 }
 
-void copyDir(String from, String to) {
-  _copyDirImpl(Directory(from), Directory(to));
+String copyDir(String from, String to) {
+  final destDir = Directory(join(to, 'ditto-export-${DateTime.now().millisecondsSinceEpoch}'));
+  destDir.createSync(recursive: true);
+  _copyDirImpl(Directory(from), destDir);
+  return destDir.path;
 }
 
 void _copyDirImpl(Directory source, Directory destination) =>
     source.listSync(recursive: false).forEach((var entity) {
```

Comment:
Making this async will require mapping to an Iterable of futures, then running them in parallel via `Future.wait`